CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](https://keepachangelog.com), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [3.0.0] - 10-04-2024
+
+### Added
+ - Release workflows and issue templates to the target folder.
+ - AddRange method added to the customList class in the debugStart.ps1.
+ - Email property to the fieldMapping.json file.
```

Comment:
Misschien nog toevoegen dat deze 'by default' is gemapped op: `MicrosoftActiveDirectory.mail`.